tests/test_repository.py

```diff
@@ -177,9 +177,37 @@ def test_repo_content_licensing(container_per_test) -> None:
     ), "Found a package with a SUSE-Firmware license"
 
 
+@pytest.mark.skipif(
+    (
+        OS_VERSION.startswith("15")
+        and OS_VERSION not in ALLOWED_BCI_REPO_OS_VERSIONS
+    ),
+    reason="no included BCI repository",
+)
```

Comment:
Are you sure this is the right condition? Because the test now runs on Tumbleweed

Reply:
now it is.